Detectors/CPV/simulation/include/CPVSimulation/CPVSimParams.h

```diff
@@ -0,0 +1,50 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_CPV_CPVSIMPARAMS_H_
+#define O2_CPV_CPVSIMPARAMS_H_
+
+#include "SimConfig/ConfigurableParam.h"
+#include "SimConfig/ConfigurableParamHelper.h"
+
+namespace o2
+{
+namespace cpv
+{
+// parameters used in responce calculation and digitization
+// (mostly used in GEANT stepping and Digitizer)
+struct CPVSimParams : public o2::conf::ConfigurableParamHelper<CPVSimParams> {
+  //Parameters used in conversion of deposited energy to APD response
+  int mnCellZ = 128;
+  int mnCellX = 60;
+  float mPadSizeZ = 1.13;         ///<  overall size of CPV active size
+  float mPadSizeX = 2.1093;       ///<  in phi and z directions
+  float mCellWr = mPadSizeX / 2.; ///<  Distance between wires (2 wires above 1 pad)
```

Comment:
there is another parameter that depend on other members. Please move this to a function.